backend/src/nodes/nodes/utility/parse_number.py

```diff
@@ -0,0 +1,32 @@
+from __future__ import annotations
+
+from ...node_base import NodeBase
+from ...node_factory import NodeFactory
+from ...properties.inputs import NumberInput, TextInput
+from ...properties.outputs import NumberOutput
+from . import category as UtilityCategory
+
+
+@NodeFactory.register("chainner:utility:parse_number")
+class ParseNumberNode(NodeBase):
+    def __init__(self):
+        super().__init__()
+        self.description = "Parses text to base-10."
+        self.inputs = [
+            TextInput("Text", min_length=0),
+            NumberInput("Base", default=10, minimum=2, maximum=36),
+        ]
+        self.outputs = [
+            NumberOutput(
+                "Value",
+                output_type="int & number::parseInt(Input0, Input1)",
```

Comment:
Woops, I forgot something.

```suggestion
                output_type="int & number::parseInt(toString(Input0), Input1)",
```